modules/programs/tarsnap.nix

```diff
@@ -0,0 +1,300 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.tarsnap;
+
+in {
+  options = {
+    programs.tarsnap = {
+      enable = mkEnableOption "Tarsnap";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.tarsnap;
+        defaultText = literalExample "pkgs.tarsnap";
+        description = ''
+          Tarsnap package to install
+        '';
+      };
+
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Extra config to add to the end of .tarsnaprc
+        '';
+      };
+
+      aggressiveNetworking = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Use multiple TCP connections to send data to the
```

Comment:
These descriptions seem copied from the tarsnap documentation. Is its license MIT compatible?

Reply:
They are. I could be misreading it, but it looks like the docs themselves are BSD 3-clause: https://github.com/Tarsnap/tarsnap/blob/master/tar/tarsnap.1-mdoc.in